packages/common/src/ui/components/popover/InfoTooltipIcon/InfoTooltipIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Tooltip } from '@mui/material';
+import { InfoIcon } from '../../../icons';
+
+export const InfoTooltipIcon = ({ title }: { title: string }) => (
+  <Tooltip title={title}>
+    <div style={{ transform: 'scale(0.7)', cursor: 'help' }}>
+      <InfoIcon fontSize="small" />
+    </div>
+  </Tooltip>
+);
```

Comment:
1. Pretty specific. Maybe better to make an `IconWithTooltip` and then compose this component from that
2. popover folder or icons or new tooltip folder?

Reply:
heh, yeah. An `IconWithTooltip` would be good and then this one could be composed from it.
As for where to put it.. I like the idea of a new tooltip folder; there's not much to put in it though.. probably just keep in popover?

Reply:
Will revisit when consolidating the user tooltip util